src/tests/test_cases/test.py

```diff
@@ -429,5 +429,5 @@ def bad_google_string():  # noqa: D400
     """Test a valid something"""
 
 
-expect(os.path.normcase(__file__ if __file__[-1] != 'c' else __file__[:-1]),
+expect(os.path.relpath(__file__),
```

Comment:
Can you explain this change? This logic was handling a subtle bug where is subsequent execution, `__file__` contains the `.pyc` version which cannot be parsed by `pydocstyle`.